docs/reference_guides/core/process_block.rst

```diff
@@ -28,32 +28,32 @@ ProcessBlockData class.
           self.y = Var(initialize=self.config.yinit)
 
 The following example demonstrates creating a scalar instance of the new class.
-The ``default`` key word argument is used to pass information on the the
-MyBlockData ConfigBlock.
+The keyword arguments are used to pass information on the the MyBlockData
+ConfigBlock.
 
 .. testcode::
 
     m = ConcreteModel()
-    m.b = MyBlock(default={"xinit":1, "yinit":2})
+    m.b = MyBlock(xinit=1, yinit=2)
 
 The next example creates an indexed MyBlock instance.  In this case, each block is
-configured the same, using the ``default`` argument.
+configured the same, using the keyword arguments.
 
-.. testcode:: 
+.. testcode::
 
     m = ConcreteModel()
-    m.b = MyBlock([0,1,2,3,4], default={"xinit":1, "yinit":2})
+    m.b = MyBlock([0,1,2,3,4], xinit=1, yinit=2)
 
-The next example uses the ``initialize`` argument to override the configuration of
-the first block. Initialize is a dictionary of dictionaries where the key of the
-top level dictionary is the block index and the second level dictionary is
-arguments for the config block.
+The next example uses the ``initialize`` argument to override the default
+configuration for block 0. Initialize is a dictionary of dictionaries where the
+key is the block index and the value is the configuration dictionary.
 
-.. testcode:: 
+.. testcode::
 
     m = ConcreteModel()
-    m.b = MyBlock([0,1,2,3,4], default={"xinit":1, "yinit":2},
-                  initialize={0:{"xinit":1, "yinit":2}})
+    m.b = MyBlock(
+        [0,1,2,3,4], xinit=1, yinit=2, initialize={0:{"xinit":1, "yinit":2}}
```

Comment:
Whilst we are here, is `initialize` the best name for this? It might well be, but this is the time to ask the question. Given that "initialize" also has a very important (and different meaning) for solving the model, I wonder if we should avoid using it here to prevent any chance of confusion,.

That said, the only ideas I have had so far are some variation on "indexed_config" or similar, which I am not sure is really any better.

Reply:
It is consistent with other promo components and they are arguments for object initialization.

Reply:
That said, it would be a heck of a lot easier to change than default.

Reply:
If we do change initialize, let's do it in another PR.